influunt-api/modules/influunt-central/app/server/conn/MQTTServerActor.java

```diff
@@ -157,15 +137,13 @@ private void connect() throws MqttException {
             sendToBroker(message);
         });
 
-        client.subscribe("central/echo", 1, (topic, message) -> {
+        client.subscribe("central/+", 1, (topic, message) -> {
```

Comment:
Sim. É um wildcard

On Wednesday, September 14, 2016, Pedro Pires <notifications@github.com>
wrote:

> In influunt-api/modules/influunt-central/app/server/conn/
> MQTTServerActor.java
> <https://github.com/influunt/influunt/pull/827#discussion_r78793932>:
>
> > @@ -157,15 +137,13 @@ private void connect() throws MqttException {
> >              sendToBroker(message);
> >          });
> >
> > -        client.subscribe("central/echo", 1, (topic, message) -> {
> > +        client.subscribe("central/+", 1, (topic, message) -> {
>
> O nome do canal (central/+) está certo?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/influunt/influunt/pull/827/files/3a51a7a61b982b4611dfea63a377e5b6b556ec0c#r78793932>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGDrQa4FrheLHBb4wEJs1Hp_GZHihbPks5qqC1OgaJpZM4J9BFO>
> .
>


-- 
Rodrigo Sol

rodrigo@rarolabs.com.br
http://www.rarolabs.com.br